ui/app/templates/components/namespace-picker.hbs

```diff
@@ -1,110 +1,84 @@
             </NamespaceLink>
           {{/unless}}
-          <h5 class="list-header">Current namespace</h5>
+          <h5 class="list-header">Namespaces</h5>
         </div>
-        <div class="namespace-header-bar level is-mobile">
-          <div class="level-left">
-            <header>
-              <div class="level is-mobile namespace-link">
-                <span class="level-left" data-test-current-namespace>
-                  {{if this.namespacePath (concat this.namespacePath "/") "root"}}
-                </span>
-                <span class="level-right">
-                  <Icon @name="check-circle" class="has-text-success" />
-                </span>
-              </div>
-            </header>
+        {{#if (includes "" this.lastMenuLeaves)}}
+          {{! leaf is '' which is the root namespace, and then we need to iterate the root leaves }}
+          <div class="leaf-panel {{if (eq '' this.currentLeaf) 'leaf-panel-current' 'leaf-panel-left'}} ">
+            {{#each this.rootLeaves as |rootLeaf|}}
+              <NamespaceLink @targetNamespace={{rootLeaf}} @class="namespace-link" @showLastSegment={{true}} />
+            {{/each}}
           </div>
-        </div>
-        <div class="namespace-list {{if this.isAnimating 'animated-list'}}">
-          <div class="is-mobile level-left">
-            {{#unless this.isUserRootNamespace}}
-              <NamespaceLink
-                @targetNamespace={{or
-                  (object-at (dec 2 this.menuLeaves.length) this.lastMenuLeaves)
-                  this.auth.authData.userRootNamespace
-                }}
-                @class="namespace-link is-current button is-ghost icon"
-              >
-                <Chevron @direction="left" @class="has-text-grey" />
-              </NamespaceLink>
-            {{/unless}}
-            <h5 class="list-header">Namespaces</h5>
-          </div>
-          {{#if (includes "" this.lastMenuLeaves)}}
-            {{! leaf is '' which is the root namespace, and then we need to iterate the root leaves }}
-            <div class="leaf-panel {{if (eq '' this.currentLeaf) 'leaf-panel-current' 'leaf-panel-left'}} ">
-              {{~#each this.rootLeaves as |rootLeaf|}}
-                <NamespaceLink @targetNamespace={{rootLeaf}} @class="namespace-link" @showLastSegment={{true}} />
-              {{/each~}}
-            </div>
-          {{/if}}
-          {{#each this.lastMenuLeaves as |leaf|}}
+        {{/if}}
+        {{#each this.lastMenuLeaves as |leaf|}}
+          {{#if leaf}}
             <div
               class="leaf-panel
                 {{if (eq leaf this.currentLeaf) 'leaf-panel-current' 'leaf-panel-left'}}
                 {{if (and this.isAdding (eq leaf this.changedLeaf)) 'leaf-panel-adding'}}
                 {{if (and (not this.isAdding) (eq leaf this.changedLeaf)) 'leaf-panel-exiting'}}
                 "
             >
-              {{~#each-in (get this.namespaceTree leaf) as |leafName|}}
```

Comment:
How did that each even work with the `~`? Does this mean something I don't know about or was this an error?

Reply:
Ya I was surprised by that too! I'm pretty sure that used to be a private helper (maybe)? Not sure why that wasn't raising a warning.